src/caselawclient/xquery/set_metadata_court.xqy

```diff
@@ -2,17 +2,33 @@ xquery version "1.0-ml";
 
 declare namespace akn = "http://docs.oasis-open.org/legaldocml/ns/akn/3.0";
 declare namespace uk = "https://caselaw.nationalarchives.gov.uk/akn";
+declare function local:delete($uri)
```

Comment:
is there a way to test xquery functions?

Reply:
I would very much like automated unit tests for XQuery; there is https://developer.marklogic.com/code/marklogic-unit-test/ but I haven't tried getting that to work yet. I have tested these manually in staging in the console.